mmpose/codecs/simcc_label.py

```diff
@@ -35,19 +35,29 @@ def __init__(self,
                  input_size: Tuple[int, int],
                  simcc_type: str = 'gaussian',
                  sigma: float = 6.0,
-                 simcc_split_ratio: float = 2.0) -> None:
+                 simcc_split_ratio: float = 2.0,
+                 label_smoothing: float = 0.0) -> None:
         super().__init__()
 
         self.input_size = input_size
         self.simcc_type = simcc_type
         self.sigma = sigma
         self.simcc_split_ratio = simcc_split_ratio
+        self.label_smoothing = label_smoothing
 
-        if self.simcc_type not in {'gaussian', 'one-hot'}:
+        if self.simcc_type not in {'gaussian', 'smoothing', 'one-hot'}:
             raise ValueError(
                 f'{self.__class__.__name__} got invalid `simcc_type` value'
                 f'{self.simcc_type}. Should be one of '
-                '{"gaussian", "one-hot"}')
+                '{"gaussian", "smoothing", "one-hot"}')
+
+        if self.simcc_type in {'gaussian', 'one-hot'
```

Comment:
Maybe we can just ignore ``label_smoothing`` if the type is not 'smoothing'? So we do not have to check equal between float values.